bedrock/firefox/tests/test_base.py

```diff
@@ -344,6 +344,36 @@ def test_fx_81_0_0(self, render_mock):
 
     # end 81.0 whatsnew tests
 
+    # begin 82.0 whatsnew tests
+
+    @patch.object(fx_views, 'ftl_file_is_active', lambda *x: True)
+    def test_fx_82_0_0(self, render_mock):
+        """Should use whatsnew-fx82 template for 82.0 in English"""
+        req = self.rf.get('/firefox/whatsnew/')
+        req.locale = 'en-US'
+        self.view(req, version='82.0')
+        template = render_mock.call_args[0][1]
+        assert template == ['firefox/whatsnew/whatsnew-fx82.html']
+
+    @patch.object(fx_views, 'ftl_file_is_active', lambda *x: True)
+    def test_fx_82_0_0_locale(self, render_mock):
+        """Should use standard whatsnew template for 82.0 in other locales"""
+        req = self.rf.get('/firefox/whatsnew/')
+        req.locale = 'es-ES'
+        self.view(req, version='82.0')
+        template = render_mock.call_args[0][1]
+        assert template == ['firefox/whatsnew/index-account.html']
+
+    def test_fx_82_0_0_china(self, render_mock):
+        """Should use standard whatsnew template in China for en-US locale"""
+        req = self.rf.get('/firefox/whatsnew/china/')
+        req.locale = 'en-US'
+        self.view(req, version='82.0')
+        template = render_mock.call_args[0][1]
+        assert template == ['firefox/whatsnew/index-account.html']
```

Comment:
I'm not sure why this test is failing:
```
E       AssertionError: assert ['firefox/wha...ew-fx82.html'] == ['firefox/what...account.html']
E         At index 0 diff: 'firefox/whatsnew/whatsnew-fx82.html' != 'firefox/whatsnew/index-account.html'
```
The view seems to be working and I get the expected page locally at /whatsnew/china/ but this test is evidently getting the fx82 template instead. I'm not sure what I'm missing, but either the test is wrong or the view is wrong.